ЛогическиеОперации_Номер_2/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace ЛогическиеОперации_Номер_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+
+        {
+            var x = Input("x");
+            var y = Input("y");
+
+            Console.WriteLine("Точка принадлежит графику " + CheckGraphic(x, y));
```

Comment:
Не очень удачное выражение. Под словом "график" обычно понимают график функции, т. е. кривую. Здесь налицо некая область.
```suggestion
            Console.WriteLine("Точка принадлежит заданной области: " + CheckGraphic(x, y));
```